src/extension/default_extension_ek.cairo

```diff
@@ -146,6 +145,7 @@ mod DefaultExtensionEK {
                 tokenization::{tokenization_component, tokenization_component::TokenizationTrait}
             }
         },
+        vendor::erc20::{ERC20ABIDispatcher, ERC20ABIDispatcherTrait},
```

Comment:
Same nitty nit dispatcher comment here 🐿️

It's too bad the reading of decimals has to be done here as well, a leak in the abstraction, but well, rather keep it consistent with how the other oracles are.